interface/forms/misc_billing_options/new.php

```diff
@@ -112,27 +74,27 @@ function genProviderSelect($selname, $toptext, $default=0, $disabled=false) {
  </tr>
  &nbsp;&nbsp;
 <tr>
- <td><span class=text><?php echo xlt('BOX 16. Date unable to work to');?>:</span></td>
+ <td><span class=text><?php echo xlt('Box 16. Date unable to work to');?>:</span></td>
   <td><?php $off_work_to = $obj{"off_work_to"}; ?>
     <input type=text style="width: 70px;" size=10 class='datepicker' name='off_work_to' id='off_work_to'
     value='<?php echo attr($off_work_to); ?>'
-    title='<?php echo xla('yyyy-mm-dd'); ?>' />
+    title='<?php echo xla('yyyy-mm-dd'); ?>'
```

Comment:
ok, i can fix this if you'd like and rebase the commits into but will need your assistance on the latter, thanks Brady

ps it's testing well